addon/components/frost-fixed-table.js

```diff
@@ -334,5 +348,20 @@ export default Component.extend({
   // == Actions ===============================================================
 
   actions: {
+    /**
+     * Wrap our arguments in a single object, so that cell renderers can trigger arbitrary events.
+     * Your handler is then responsible for doing stuff based on actions like 'click' or 'input'.
+     *
+     * @param {Number} row - data rows are zero based, and header has row -1
+     * @param {Number} col - column index
+     * @param {String} action - this comes after row/col as Ember lets us include those in action closures easily.
+     * @param {Object[]} args - any additional data
+     */
+    handleCallback (row, col, action, args) {
+      const handler = this.get('onCallback')
+      if (handler) {
```

Comment:
You shouldn't need to do this but you can also do:
```js
if (typeOf(handler) === 'function') {
  handler({action, args, col, row})
}
```
if you want type checking here to ensure `handler` is a function before trying to make a call of it.